datafusion-cli/src/functions.rs

```diff
@@ -376,16 +397,11 @@ impl TableFunctionImpl for ParquetMetadataFunc {
                 let converted_type = column.column_descr().converted_type();
 
                 if let Some(s) = column.statistics() {
-                    let (min_val, max_val) = if s.has_min_max_set() {
-                        let (min_val, max_val) =
-                            convert_parquet_statistics(s, converted_type);
-                        (Some(min_val), Some(max_val))
-                    } else {
-                        (None, None)
-                    };
+                    let (min_val, max_val) =
+                        convert_parquet_statistics(s, converted_type);
```

Comment:
min() and max() are deprecated, so this updates the code to use `min_opt()` and `max_opt()` which I think makes this much less convoluted